src/Resources/views/Form/div_layout.html.twig

```diff
@@ -1,77 +1,107 @@
 {% block ace_editor_widget %}
     {% apply spaceless %}
-    <textarea {{ block('widget_attributes') }}>{{ value }}</textarea>
-    <div id="{{ id }}_ace" {% for attrname, attrvalue in wrapper_attr %}{% if attrname == 'title' %}{{ attrname }}="{{ attrvalue|trans({}, translation_domain) }}" {% else %}{{ attrname }}="{{ attrvalue }}" {% endif %}{% endfor %}></div>
-    {{ include_ace_editor() }}
-    <script type="text/javascript">
-    (function () {
-        var textarea = document.getElementById('{{ id }}'),
-            editorElm = document.getElementById('{{ id }}_ace'),
-            editor = ace.edit(editorElm),
-            width = {{ width.value }},
-            widthUnit = '{{ width.unit }}',
-            height = {{ height.value }},
-            heightUnit = '{{ height.unit }}';
+		{% if use_stimulus %}
+			<div {{ stimulus_controller('norberttech/aceeditor-bundle/ace',{
```

Comment:
path

Reply:
I dont know what is wrong whit this. this is again name from package.json.